### PR TITLE
[Feature] import-weights command

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Add `import-weight` command that pulls weight from a gSheet and populate the DB

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

N/A

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/signals.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        )
+        instance.country = country
+        instance.print_weight = print_weight
+        instance.radio_weight = radio_weight
+        instance.tv_weight = tv_weight
+        instance.internet_weight = internet_weight
+        instance.twitter_weight = twitter_weight
+        instance.save()
+    except (ObjectDoesNotExist, KeyError):
+        pass
```
Comment (esirK) [inline reports/signals.py:44]: shouldn't we at least raise the error for sentry to notify us?

**Review by esirK — APPROVED** [review]
🚀

File: reports/signals.py (hunk 1 above)
Comment (kilemensi) [reply reports/signals.py:44]: Look at him making me do the janitorial work! All current `gsheets` signals don't raise exception! :sigh:

Comment (kilemensi) [reply reports/signals.py:44]: ... aaand GMMP doesn't have Sentry integration.

Comment (esirK) [reply reports/signals.py:44]: cool cool. Will do that latter then

**Pull request merged**